src/docs/ftw-availability/availability-management/index.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Managing listing availability
+slug: availability-management
+updated: 2025-03-24
+category: template-availability
+ingress:
+  Learn about timezones and availability management in the Template.
+published: true
+---
+
+This article explains how listing availability is handled in the
+Sharetribe Web Template, with a focus on technical details and
+implementation. For a general overview of availability management, see
+our
+[Help Center](https://www.sharetribe.com/help/en/articles/8413212-how-availability-management-works).
+See also the
+[reference documentation for availability management](<(/references/availability)>).
+
+## Availability plan types and timezone support
+
+The API supports creating availability plans with either type
+`availabilityPlan/day` with day-level availability or type
+`availabilityPlan/time` with time range level availability. When a user
+creates a listing with the Sharetribe Web Template, the listing is
+always created with type `availabilityPlan/time`, whether the calendar
+booking type is daily, nightly, hourly or fixed.
+
+Using the `availabilityPlan/time` type means that the template can
+specify a certain time zone for the listing's availability plan, which
+is asked when creating a bookable listing. The timezone is associated
+with the availability plan and ensures that all booking times are
+interpreted according to the local time of the listing, rather than the
+time zone of the customer making the booking.
```

Comment:
```suggestion
time zone of the customer making the booking - or UTC time zone, which `availabilityPlan/day` uses internally.
```

Reply:
@Gnito When is UTC used instead of customer time zone?

Reply:
@shareoc to illustrate availabilityPlan/day a bit:

If someone tried to book a listing on New Zealand (2.4.2025 - 4.4.2025) on top of FTW-daily,
what was actually booked was 00:00 moments on UTC, which translates as
2.4.2025 klo 13.00.00 - 4.4.2025 klo 13.00.00 (local time is 13 hours ahead of UTC)